specification/cognitiveservices/data-plane/UnifiedVision/search/preview/2024-03-01-preview/UnifiedVision-Search.json

```diff
@@ -0,0 +1,1227 @@
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetIndex_MaximumSet_Gen": {
+            "$ref": "./examples/Retrieval_GetIndex.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "IngestionApi"
+        ],
+        "summary": "Updates an index.",
+        "description": "Updates an index with the specified name.",
+        "operationId": "UpdateIndex",
+        "consumes": [
+          "application/json-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "indexName",
+            "description": "The name of the index to be updated.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The request body containing the updates to be applied to the index.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateIngestionIndexRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/GetIngestionIndexResponse"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "UpdateIndex_MaximumSet_Gen": {
+            "$ref": "./examples/Retrieval_UpdateIndex.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "IngestionApi"
+        ],
+        "summary": "Deletes an index.",
+        "description": "Deletes an index and all its associated ingestion documents.",
+        "operationId": "DeleteIndex",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "indexName",
+            "description": "The name of the index to be deleted.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          }
+        },
+        "x-ms-examples": {
+          "DeleteIndex_MaximumSet_Gen": {
+            "$ref": "./examples/Retrieval_DeleteIndex.json"
+          }
+        }
+      }
+    },
+    "/retrieval/indexes": {
+      "get": {
+        "tags": [
+          "IngestionApi"
+        ],
+        "summary": "Retrieves all indexes.",
+        "description": "Retrieves a list of all indexes across all ingestions.",
```

Comment:
As you can see, one of the checks is spell check. It is complaining about the word 'ingestions'. Please put it and any other words you see in the spell check in to the custom-words.txt file present at the root of this repo.

Reply:
added